docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+status: Draft
+authors: 
+---
+
+# Dapp browser API usage
+    
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Implementation](#implementation)
+4. [Usage](#usage)
+    * [Properties](#properties)
+        * [`isStatus`](#isStatus)
+        * [`status`](#status)
+    * [Methods](#methods)
+        * [`enable`](#enable)
+        * [`isConnected`](#isConnected)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+Status exposes an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. It also provides bidirectional communication between JS code and Status.
+
+## Definitions
+
+| Term       | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| **Webview**   | React Native wrapper for a platform-specific browser core implementation.                                        |
+| **Ethereum Provider** | A JS object (`window.ethereum`) injected into each web page opened in the browser providing web3 support. |
+| **Bridge** | A set of facilities allow bidirectional communication between JS code and Status. |
+
+## Implementation
+Status uses a [forked version](https://github.com/status-im/react-native-webview) of [react-native-webview](https://github.com/react-native-community/react-native-webview)  to display web or dapps content. The fork provides an Android implementation of JS injection before page load. It is required in order to properly inject Ethereum Provider object.
```

Comment:
Any chance this can be upstreamed?

Reply:
Also do we have a method to keep up with upstream changes?

Reply:
Haven't yet updated our fork with upstream changes. I did rebase with latest upstream locally some time ago, but didn't tag it yet. Changes in our fork affect just a single file, so it is simple.

I've added a comment in their repo https://github.com/react-native-community/react-native-webview/pull/1099#issuecomment-611215560, but it's been quiet yet. Optimally - yes, we'd push a PR to their repo and close our fork.